distribution/docs/src/main/resources/content/_managing/_configuring/web-context-policy-manager.adoc

```diff
@@ -47,6 +52,20 @@ Any web context that allows or requires specific authentication types should be
 
 |===
 
+==== Terminating Authentication Types
+Terminating authentication types are authentication types where, once triggered, must either obtain credentials or forbid access to the system.
+No succeeding configured authentication types will be used after a terminating authentication type is triggered.
+The terminating auth types are `BASIC`, `SAML`, and `OIDC`.
```

Comment:
This isn't 100% true. SAML and OIDC will fall through if none of the conditions are met. For example, if the client is not a browser and doesn't support ECP, then it would fall through to whatever is after it. So SAML|BASIC is still viable, for instance.